source/iostreams.tex

```diff
@@ -9588,7 +9588,7 @@
       void str(const T& t);
 
   private:
-    basic_stringbuf<charT, traits> sb;  // \expos
+    basic_stringbuf<charT, traits, Allocator> sb;  // \expos
```

Comment:
```suggestion
    basic_stringbuf<charT, traits, Allocator> sb;   // \expos
```

Reply:
@xmcgcg , please fix this (we check indentation of comments, and this issue causes our regression tests to fail).

Reply:
Suggestion commited :)